Src/FluentAssertions/Common/Guard.cs

```diff
@@ -63,6 +63,22 @@ public static void ThrowIfArgumentIsEmpty<T>(IEnumerable<T> values, string param
         }
     }
 
+    public static void ThrowIfArgumentIsEmpty(string str, string paramName, string message)
+    {
+        if (str.Length == 0)
```

Comment:
I'd suggest using constant pattern matching for constant comparisons in new code:
```suggestion
        if (str.Length is 0)
```

Reply:
I think this is the reason:
>  using constant pattern matching